typescript/sdk/src/warp/types.ts

```diff
@@ -59,4 +38,29 @@ export const WarpCoreConfigSchema = z.object({
     .optional(),
 });
 
+// List of constant values for local or interchain fees
+export type FeeConstantConfig = z.infer<typeof FeeConstantConfigSchema>;
+
+// List of chain pairs to blacklist for warp routes
+export type RouteBlacklist = Array<{
+  origin: ChainName;
+  destination: ChainName;
+}>;
+
+// Transaction types for warp core remote transfers
+export enum WarpTxCategory {
+  Approval = 'approval',
+  Transfer = 'transfer',
+}
+
+export type WarpTypedTransaction = TypedTransaction & {
+  category: WarpTxCategory;
+};
```

Comment:
you really don't like the array approach :P

Reply:
Like I said, the UI _needs_ to know this context :)